menu-bridges

```diff
@@ -271,31 +259,48 @@ if [ $exitstatus = 0 ]; then
       read -n 1 -s -r -p "Press any key to continue"
 
 # Add bridges
-    elif [ $CHOICE = 4 ]; then
+    elif [ $CHOICE = 3 ]; then
       sudo bash bridges_add_old $MODE_BRIDGES
       read_config
 
 # Activate bridges
-    elif [ $CHOICE = 5 ]; then
+    elif [ $CHOICE = 4 ]; then
       if [ "$MODE_MEEK" = "Bridge meek_lite " ] || [ "$MODE_SNOW" = "Bridge snowflake " ]; then
         whiptail --title "TorBox - INFO" --textbox text/no_meek-snow-please-text $MENU_HEIGHT_15 $MENU_WIDTH_REDUX
       else
-        sudo bash bridges_activate_old 1
+        if [ "$MODE_BRIDGES" != "UseBridges 1" ]; then
+          INPUT=$(cat text/activate-bridges-text)
+          if (whiptail --title "TorBox - INFO (scroll down!)" --scrolltext --defaultno --no-button "NO" --yes-button "YES" --yesno "$INPUT" $MENU_HEIGHT_25 $MENU_WIDTH); then
+            deactivating_bridge_relay
+            sudo bash bridges_activate_old $configured_bridges_deactivated $number_configured_bridges_deactivated 0
+            if [ $number_configured_bridges_total = 0]; then
+              INPUT=$(cat text/add-bridges-first-text)
+              if (whiptail --title "TorBox - INFO" --yesno "$INPUT" 35 $MENU_WIDTH); then
+                sudo bash bridges_add_old
```

Comment:
This is the use case regarding this https://github.com/radio24/TorBox/pull/56#discussion_r648516091
Cause if there is no bridges it, not even deactivated, it will ask bridges_add_old to add bridges, to this should trigger bridges config lines `UseBridges` to be activated.